templates/citizen_action_manager/cancel.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'citizen_action/_layout.html.twig' %}
+
+{% block page_title citizen_action.name %}
+
+{% block citizen_action_content %}
+    <section class="l__wrapper--slim space--30-0 text--body">
+        <div class="b__nudge--bottom-huge">
+            <h2 class="b__nudge--bottom-large text--medium">
+                Annuler l'action citoyenne
+            </h2>
+            <div class="text--body">
+                Souhaitez-vous réellement annuler l'action citoyenne {{ citizen_action.name }} ?
+            </div>
+            <div class="text--body">
+                Cette action est irréversible et enverra un e-mail d'annulation à tous les inscrits.
+            </div>
+
+            {{ form_start(form, { attr: { class: 'b__nudge--top-40'} }) }}
+
+            {{ form_errors(form) }}
```

Comment:
I prefer when the code is indented between `form_start` and `form_end`. What is the standard on this project ? Not sure